database/mssql/scripts/dbscripts/PSP_PIMS_S25_00/Build/25_PIMS_CONTACT_MGR_VW_Fix.SQL

```diff
@@ -0,0 +1,90 @@
+/* Drop and recreate the PIMS_CONTACT_MGR_VW view to include the subselect in 
```

Comment:
I'm nervous as to why this file is in this PR, it seems unrelated to the other changes added to this PR.